lib/frame.js

```diff
@@ -235,7 +235,8 @@ api.frame = (state, subjects, frame, parent, property = null) => {
         // recurse into list
         if(graphTypes.isList(o)) {
           const subframe = (frame[prop] && types.isObject(frame[prop][0]) ?
-            frame[prop][0]['@list'] : _createImplicitFrame(flags));
+            (frame[prop][0]['@list'] || _createImplicitFrame(flags)) :
+            _createImplicitFrame(flags));
```

Comment:
Logic can be compacted, shift `@list` check in to the conditional:
```suggestion
          const subframe = (frame[prop] && types.isObject(frame[prop][0] && frame[prop][0]['@list']) ?
￼            frame[prop][0]['@list'] : _createImplicitFrame(flags));
```

Reply:
Fixed in faaaf6a01db684ef2871343c989a0d97849e3a5b.